Multisig/Cross-layer/App.swift

```diff
@@ -8,10 +8,12 @@
 
 import Foundation
 
-class App {
+class App: ObservableObject {
 
     static let shared = App()
-
+    
+    @Published
+    var state: ViewSate = .balanaces
```

Comment:
nice idea!

I would improve on it by extracting this tab selection state into a separate observable object, and then assigning it as environment object to the content view, instead of using this shared app class.

Reply:
But this object should have a lifetime of the app, so it will be a singleton anyways, or do you have smth other in mind?

Reply:
this will live in the App, like a relay service